data-raw/process_data_set_rana.R

```diff
@@ -0,0 +1,16 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+# mlb-quasi-win-shares
+quasi_winshares <- read_csv("data-raw/mlb-quasi-win-shares/quasi_winshares.csv") %>%
+  clean_names()%>%
+  mutate(
+    team_id = as.factor(team_id),
+    lg_id   = as.factor(lg_id),
+    def_pos = as.factor(def_pos)
```

Comment:
Could you 
- convert `franch_id`, `prev_franc`, and `next_franch` to factors as well
- For `prev_franc` and `next_franch`, can you replace all `NULL` with `NA` (missing value) and update the help file accordingly?
- For `def_pos` (defensive position), it's going to be hard to work with this variable as is. For example, look at the first row for Ketel Morse `"CF, 2B, SS"`. Could you instead create 9 boolean `TRUE`/`FALSE` variables: `P`, `C`, `1B`, `2B`, `3B`, `SS`, `LF`, `CF`, `RF` and update help file accordingly? Ask @beanumber for help on this if you don't know how